ui/src/containers/NodeInformation.js

```diff
@@ -189,6 +194,10 @@ const NodeInformation = props => {
     };
   });
 
+  const searchedVolume = volumeData.filter(volume =>
+    volume.name.includes(localStorage.getItem(SET_SEARCHED_VOLUME)),
```

Comment:
localStorage.getItem(SET_SEARCHED_VOLUME) should be replaced by a redux state. You retrieve it on the start-up of the application and put it in a redux state. you update the redux state and the localstorage on search's onChange

Reply:
Actually we don't need to store the searched name in localStroage. Store it in the redux-store in enough.